packages/web/src/brandkit/Intro.tsx

```diff
@@ -29,11 +33,44 @@ const Overview = withNamespaces(NameSpaces.brand)(function _Overview({ t }: I18n
       <H1 style={standardStyles.elementalMarginBottom}>{t('home.title')}</H1>
       <H4 style={standardStyles.blockMarginBottom}>{t('home.introduction')}</H4>
       <Text style={[fonts.h5, standardStyles.elementalMarginBottom]}>{t('home.useageTitle')}</Text>
-      <Text style={fonts.p}>
+      <Text style={[fonts.p, standardStyles.elementalMarginBottom]}>
         <Trans ns={NameSpaces.brand} i18nKey="home.useageText">
           <InlineAnchor href={menuItems.CODE_OF_CONDUCT.link}>Code Of Conduct</InlineAnchor>
         </Trans>
       </Text>
+
+      <Text style={[fonts.h5, standardStyles.elementalMargin]}>{t('home.importantRemember')}</Text>
+      <Text style={fonts.p}>
+        {t('home.celoOwner')}
+        <Ul>
+          <Trans ns={NameSpaces.brand} i18nKey={'home.celoOwnerBullets'}>
+            <Li>{}</Li>
+            <Li>{}</Li>
+          </Trans>
+        </Ul>
+      </Text>
     </View>
   )
 })
+
+function BrandVoice() {
+  const { t } = useTranslation(NameSpaces.brand)
+
+  return (
+    <View style={brandStyles.gap}>
+      <SectionTitle>{t('home.brandVoice')}</SectionTitle>
+      {/* <Text style={[fonts.h5, standardStyles.elementalMarginBottom]}>{t('home.brandVoice')}</Text> */}
+      <Text style={fonts.p}>
```

Comment:
Wonder if we could just make a <P> tag. Not sure how often you use this, up to you

Reply:
on the fence